google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1747,6 +1747,22 @@ def default_acl
         # @param [Boolean] force [Deprecated] Force the latest policy to be
         #   retrieved from the Storage service when `true`. Deprecated because
         #   the latest policy is now always retrieved. The default is `nil`.
+        # @attr [Integer] requested_policy_version The requested syntax schema version of
+        #   the policy. Optional. If `1`, `nil` or not provided, `Policy#roles` and related
+        #   helpers will be accessible, but attempts to call `Policy#bindings` will raise
+        #   a runtime error. If `3` is provided, `Policy#bindings` will be accessible,
+        #   but attempts to call `Policy#roles` and related helpers will raise a runtime
+        #   error. A higher version indicates that the policy contains role
+        #   bindings with the newer syntax schema that is unsupported by earlier versions.
+        #
+        #   The following requested policy versions are valid:
+        #
+        #   * 1 - The first version of Cloud IAM policy schema. Supports binding one
+        #     role to one or more members. Does not support conditional bindings.
+        #   * 3 - Introduces the condition field in the role binding, which further
+        #     constrains the role binding via context-based and attribute-based rules.
+        #     See [Conditions Overview](https://cloud.google.com/iam/docs/conditions-overview)
+        #     for more information.
```

Comment:
Add https://cloud.google.com/iam/docs/policies in this section as well.

Reply:
Done in 3 locations.